spyder/widgets/mixins.py

```diff
@@ -694,13 +694,10 @@ def get_line_separator(self):
     def get_text_with_eol(self):
         """Same as 'toPlainText', replace '\n'
         by correct end-of-line characters"""

```

Comment:
```suggestion
        """
        Same as 'toPlainText', replacing '\n' by correct end-of-line
        characters.
        """
```

The format of this docstring was really awful.